tensilelite/Tensile/SolutionStructs.py

```diff
@@ -1954,6 +1958,34 @@ def assignDerivedParameters(state):
       state["SynchronizerSizeCheck"] = 1
     #   state["BatchSizeEqual"] = 1
 
+    isa = tuple(state["ISA"])
+    
+    if state["StreamK"] != 0:
+      if state["MIWaveGroup"][0] * state["MIWaveGroup"][1] != 4:
+        reject(state, "Stream-K requries MIWaveGroup0*MIWaveGroup1=4")
+      if state["EnableMatrixInstruction"] and globalParameters["AsmCaps"][isa]["HasWMMA"]:
+        reject(state, "Stream-K untested with WMMA")
+      if state["GlobalSplitU"] > 0:
+        reject(state, "Cannot enable both Stream-K and GSU")
+      # if state["PersistentKernel"]:
+      #   reject(state, "Cannot enable both Stream-K and PersistentKernel")
+      if not state["ProblemType"]["StridedBatched"]:
+        reject(state, "General batch not supported with Stream-K")
```

Comment:
This is strided batched, not general batch.

Reply:
I think the line is ok, it's checking if not strided, which is general batch in Tensile. But I don't think general batch is supported in tensilelite, so maybe better to just remove this check?

Reply:
I think I misunderstood this part.
We can keep this.